src/app/files.js

```diff
@@ -107,6 +107,47 @@ function Files (storage) {
     return files
   }
 
+  //
+  // Tree model for files
+  // {
+  //   'a': { }, // empty directory 'a'
+  //   'b': {
+  //     'c': {}, // empty directory 'b/c'
+  //     'd': { '/readonly': true, '/content': 'Hello World' } // files 'b/c/d'
+  //     'e': { '/readonly': false, '/path': 'b/c/d' } // symlink to 'b/c/d'
```

Comment:
> when updating the structure, it's important to keep /readonly and /mode in sync.

We do not yet have `/mode`. If we implement it, we may opt to remove `/readonly`.

> when updating the structure, it's important to take care, that a node does not have /content and /path

Likewise, `/path` (symlink) is not supported yet, so we're free to design it to fit.

I don't think currently `/readonly` would be valid anywhere, but leaf content nodes.